addons/killers/functions/fnc_initKillersStashes.sqf

```diff
@@ -27,7 +27,8 @@ private _createStatshesCount = ceil (random [_stashesCount/2, _stashesCount/1.33
 for "_y" from 0 to _createStatshesCount step 1 do {
     private _stash = _stashes deleteAt (floor (random (count (_stashes))));
     private _stashPos = getPos _stash;
```

Comment:
`getPos` is a bad command as there's no setter for this position type and it's slow. `getPosATL` should'be been used here. While in a building getPos will return Z = 0 while createVehicle expects getPosATL, so it will spawn the object in the floor/below the `_stash` position.

Reply:
I've started #95 to fix that project-wide.